netbox_onboarding/templates/netbox_onboarding/onboardingtask.html

```diff
@@ -0,0 +1,91 @@
+{% extends 'base.html' %}
+{% load buttons %}
+{% load custom_links %}
+{% load helpers %}
+{% load static %}
+{% load tz %}
+
+{% block header %}
+    <div class="row noprint">
+        <div class="col-sm-8 col-md-9">
```

Comment:
Given this is the only column in this row, why these specific widths rather than just `class="col"`?

Reply:
Fixed